css/css-scroll-snap/scroll-target-padding-002.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<meta charset="utf-8">
+<title>focus() and scroll-snap-padding with snapping off (y-axis)</title>
+<link rel='author' title='Elika J. Etemad' href='http://fantasai.inkedblade.net/contact'>
+<link rel='help' href='https://www.w3.org/TR/css-scroll-snap-1/#scroll-margin'>
+<link rel='match' href='scroll-target-001-ref.html'>
```

Comment:
An empty flags meta is a better way to say "no flags" than leaving it out, which might mean "no flags", or might mean "I forgot to consider flags". Not a show stopper, but I'd suggest you fix it.

Reply:
I'm going to suggest we never ask authors to do that. There's enough template to memorize already.

**It should be possible for an engineer who doesn't write reftests everyday to have the reftest template memorized.** The more requirements you pile on the less likely this is going to be.